docs/sources/features/datasources/graphite.md

```diff
@@ -89,6 +86,18 @@ being displayed in your dashboard.
 Checkout the [Templating]({{< relref "reference/templating.md" >}}) documentation for an introduction to the templating feature and the different
 types of template variables.
 
+Graphite 1.1 introduced tags and Grafana added support for Graphite queries with tags in version 5.0. To create a variable using tag values, then you need to use the Grafana functions `tags` and `tag_values`.
+
+Query | Description
+------------ | -------------
+*tags()* | Returns all tags.
+*tags(server=~backend\*)* | Returns only tags that occur in series matching the filter expression.
```

Comment:
Can you write similar to prometheus query language and use `"` with regex like `tags(server=~"^backend.*\d{2}")`? Think it can be worth pointing out in that case, maybe below the table.